oidc-authentication.md

```diff
@@ -63,6 +63,183 @@
+resides.
+
+Similar to the OAuth 2 and
+[UMA 2](https://docs.kantarainitiative.org/uma/wg/rec-oauth-uma-grant-2.0.html)
+concept of "resource owner", the resource controller MAY be an [end-user]
+(natural person) or MAY be a non-human entity treated as a person for limited
+legal purposes (legal person), such as a corporation.
+
+Note the use of "controller" as opposed to "owner"; this specification (and
+Solid in general), has no notion of owner.
+
+A given resource can have zero, one, or multiple resource controllers.
+
+##### protected resource
+A resource to which access is constrained by an appropriate authorization
+mechanism (such as Web Access Control [WAC] rules or Authorization Capabilities
+[zCAP]).
+
+##### client
+An application, service, bot, or other software entity interacting with a
+protected resource hosted by a [resource server] on behalf of a [requesting
+party]. The term "client" does not imply any particular implementation
+characteristics (such as whether the application executes on a server, a
+desktop, inside a web browser, or on other devices).
+
+The client is an [entity] in its own right, possessing an [identifier], and in
+some cases, a corresponding [controller document] containing client metadata.
+
+##### requesting party
+An [entity] (natural or legal person) that uses a [client] to seek access to a
+[protected resource]. The requesting party may or may not be the same party as
+the [resource controller].
+
+##### Solid pod (personal online datastore)
+A server compliant with the [Solid Specification]. For the purposes of
+[authentication], a pod acts as a peer node that acts according to one
+or more roles depending on the situation: as a [resource server] (RS), a
+[client]/[relying party], or an OpenID Identity Provider (OP).
+
+Because Solid uses an identity-based access control system ([WAC]), the access
+control process requires a combination of authenticated user and authenticated
+client. As a result, all Solid [resource servers] are also [relying parties].
+
+##### resource server (RS)
+As defined by [OAuth2]:
+
+> The server hosting the protected resources, capable of accepting and
+> responding to protected resource requests
+
+##### protected resource request
+A request by a [client] to a [protected resource], carrying [authentication] or
+authorization [credentials] such as an OAuth 2 Access Token, a Verifiable
+Credential, or Proof of Possession token.
+
+##### relying party (RP)
+As defined by [OIDC]:
+
+> OAuth 2.0 Client application requiring End-User Authentication and Claims
+> from an OpenID Provider.
+
+The [client] needs [authentication] related [claims]/[credentials] for UI reasons
+(to display a user name and icon to indicate that they’re logged in, for
+example), but also in its capacity as a [requesting party] to present those
+credentials to the [resource server] when making a [protected resource
+request]).
+
+##### OpenID Provider (OP)
+(Also known as an Identity Provider (IdP)).
```

Comment:
I've seen it also referred to as the Authorization Server (AS) in the spec as well

Reply:
Have this read as (Also known as the Identity Provider (IdP) or the Authorization Server (AS))

The assumption is that this parenthetical is to be used to describe other ways to refer to an OP (like how OAuth.xyz (https://oauth.xyz) uses Authorization Server) We should only use OP within this spec

Reply:
Thanks. Added the AS definition.